xdsl/dialects/riscv.py

```diff
@@ -137,11 +140,28 @@ class LiOp(RdImmOp):
     name = "riscv.li"
 
 
+# Logical
+
+
+@irdl_op_definition
+class XOrOp(RdRsRsOperation):
```

Comment:
Maybe a simple convention like *capitalize only the first character* would be a bit more straightforward? CC @AntonLydike 

```suggestion
class XorOp(RdRsRsOperation):
```

Reply:
Sounds good

Reply:
imo we should have a very simple naming scheme and not get too fancy with capitalization?